adam/visualization/make_scenes.py

```diff
@@ -749,12 +747,16 @@ def objects_to_freeze(
     Returns: ImmutableSet of object handles which are to be frozen in place by positioning model
 
     """
+
     if semantics_situation is None:
         return immutableset([])
     frozen_objects = []
     agent: Optional[SituationObject] = None
     theme: Optional[SituationObject] = None
     goal: Optional[Union[SituationObject, Region[SituationObject]]] = None
+    push_goal: Optional[Union[SituationObject, Region[SituationObject]]] = None
+    push_surface: Optional[SituationObject] = None
+    roll_surface: Optional[SituationObject] = None
```

Comment:
These edits (covering issue #732 ) got snuck in here when I thought it might help the positioning model to have less to work with (via some elements being frozen in a 2nd frame)